pkgs/servers/monitoring/prometheus/dcgm-exporter/default.nix

```diff
@@ -1,9 +1,9 @@
-{ lib
-, buildGoModule
-, fetchFromGitHub
-, cudaPackages
-, dcgm
-, linuxPackages
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  cudaPackages,
+  dcgm,
```

Comment:
I could have missed a memo somewhere. I'm curious why this style change is here.